docs/windows-design.md

```diff
@@ -56,9 +57,9 @@ containers in the same Pod to ensure that the network configuration can be corre
 operation is to make sure the DNS configuration is readable from all containers).
  
 One OVS internal port with the same name as the HNS Endpoint is also needed, in order to handle
-container traffic with OpenFlow rules. OpenFlow entries are installed to implement Pod-to-Pod and
-Pod-to-external connectivity. Service traffic is handled by kube-proxy, and a specific
-OpenFlow rule is installed to output these packets to the gateway port.
+container traffic with OpenFlow rules. OpenFlow entries are installed to implement Pod-to-Pod,
+Pod-to-external and Pod-to-ClusterIP-Service connectivity. NodePort Service traffic is handled
+by kube-proxy, and the traffic is then forwarded to local backend Pod by OVS.
```

Comment:
I think we could remove the last sentence from "NodePort Service trafficis handled..."

Reply:
Done.